apps/client/ui/Headings/FilterHeading.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC } from 'react'
+import clsx from 'clsx'
+
+type FilterHeadingProps = {
+  children: string
+  className?: string
+}
+
+export const FilterHeading: FC<FilterHeadingProps> = ({
```

Comment:
try to keep names of components the same as names of files - the best if the names everywhere (component, story and folder) are consist and clear 😀